packages/astro/src/Image.astro

```diff
@@ -0,0 +1,10 @@
+---
+import { transformProps, UnpicImageProps } from "@unpic/core";
+
+interface Props extends UnpicImageProps<any> {
+}
+
+const props = Astro.props;
```

Comment:
```suggestion
import type { HTMLAttributes } from 'astro/types'

type Props = UnpicImageProps<HTMLAttributes<'img'>> 

const props: Props = Astro.props;
```

The generic here is designed to use the framework's `<img>` type. Setting it like this will mean Astro correctly types those props

Reply:
See https://docs.astro.build/en/guides/typescript/#built-in-html-attributes